frontend/src/metabase/admin/settings/auth/constants.ts

```diff
@@ -22,7 +21,6 @@ export const GOOGLE_SCHEMA = Yup.object({
 });
 
 export const LDAP_SCHEMA = Yup.object({
-  ...PLUGIN_LDAP_FORM_FIELDS.formFieldsSchemas,
```

Comment:
we don't need to do this whole plugin dance just for the sake of not mentioning this key.